tests/components/justnimbus/test_config_flow.py

```diff
@@ -18,28 +19,26 @@ async def test_form(hass: HomeAssistant) -> None:
     assert result["type"] == FlowResultType.FORM
     assert result["errors"] is None
 
-    with patch("justnimbus.JustNimbusClient.get_data"), patch(
-        "homeassistant.components.justnimbus.async_setup_entry",
-        return_value=True,
-    ) as mock_setup_entry:
-        result2 = await hass.config_entries.flow.async_configure(
-            result["flow_id"],
-            {
-                CONF_CLIENT_ID: "test_id",
-            },
-        )
-        await hass.async_block_till_done()
-
-    assert result2["type"] == FlowResultType.CREATE_ENTRY
-    assert result2["title"] == "JustNimbus"
-    assert result2["data"] == {
-        CONF_CLIENT_ID: "test_id",
-    }
-    assert len(mock_setup_entry.mock_calls) == 1
 
```

Comment:
This is the normal flow without side effects, that should still be tested

Reply:
Oops, removed it because I wanted to reuse it -_-